nemo/collections/llm/evaluation/base.py

```diff
@@ -167,7 +169,7 @@ def generate_until(self, inputs: list[Instance]):
         return results
 
 
-def wait_for_rest_service(rest_url, max_retries=60, retry_interval=2):
+def wait_for_rest_service(rest_url, max_retries=600, retry_interval=2):
```

Comment:
could you explain why the max retries need to be increased 10x? if increasing the number of retries, are there downstream dependencies regarding expectations for how long it will take for a service to initialize?

Reply:
In my experience the TRTLLM server always took more than 1 minute to be ready. 10 minutes is just a conservative default that I decided to use.

Reply:
I agree that it's better to keep a conservative defualt. considering PyTriton takes some time to be ready. If 10 min is too large, maybe we can bring it down to 5 min or so. This should cover the base.

Reply:
Cool I can add it back.

Reply:
I think 10 is good for now as anyways it will proceed once the server is ready.